lib/lograge/sql/extension.rb

```diff
@@ -30,7 +30,13 @@ def sql(event)
   end
 end
 
-Lograge::RequestLogSubscriber.prepend Lograge::Sql::Extension
+if defined?(Lograge::RequestLogSubscriber)
+  Lograge::RequestLogSubscriber.prepend Lograge::Sql::Extension
+else
+  Lograge::LogSubscribers::ActionController.prepend Lograge::Sql::Extension
+end
+
+Lograge::LogSubscribers::ActionController.prepend Lograge::Sql::Extension
```

Comment:
This line is duplicated and probably needs to be removed

Reply:
Indeed, removed.